From 15c12251eff902fd359a0029239f957694953363 Mon Sep 17 00:00:00 2001
From: Val Packett <val@packett.cool>
Date: Sun, 5 Oct 2025 21:37:26 -0300
Subject: [PATCH] clk: qcom: gcc-x1e80100: Enable runtime PM

Enable the main clock controller driver to participate in runtime
power management.

Signed-off-by: Val Packett <val@packett.cool>
---
 drivers/clk/qcom/gcc-x1e80100.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/qcom/gcc-x1e80100.c b/drivers/clk/qcom/gcc-x1e80100.c
index 301fc9fc32d8e..96bb604c63780 100644
--- a/drivers/clk/qcom/gcc-x1e80100.c
+++ b/drivers/clk/qcom/gcc-x1e80100.c
@@ -6721,6 +6721,7 @@ static const struct qcom_cc_desc gcc_x1e80100_desc = {
 	.num_resets = ARRAY_SIZE(gcc_x1e80100_resets),
 	.gdscs = gcc_x1e80100_gdscs,
 	.num_gdscs = ARRAY_SIZE(gcc_x1e80100_gdscs),
+	.use_rpm = true,
 };
 
 static const struct of_device_id gcc_x1e80100_match_table[] = {
